stanza/text/vocab.py

```diff
@@ -134,33 +110,53 @@ def update(self, words):
         """
         return [self.add(w) for w in words]
 
-    def words2indices(self, words):
-        """
-        Convert a list of words into a list of indices.
+    def word2index(self, w):
+        return self[w]
 
-        :param words: an iterable of words to map to indices.
+    def index2word(self, i):
+        return self._index2word[i]
+
+    def freeze(self):
+        return FrozenVocab(self)
+
+    def count(self, w):
+        """Get the count for a word.
 
-        :return: the corresponding indices for each word. If a word is not found in the vocabulary then the unknown index will be returned for it.
+        :param w: a string
         """
-        return [self[w] for w in words]
+        return self._counts[w]
 
-    def indices2words(self, indices):
+    def counts_copy(self):
```

Comment:
I guess my goal was to make there be only one way to modify a `Vocab` object: through the `add` method (I think that is the case now). The old `counts` property was an exception to that. Maybe I just don't trust users enough :)